src/app/modules/project/modal-highlight/modal-highlight.component.ts

```diff
@@ -87,6 +90,10 @@ export class ModalHighlightComponent {
       this.dateErrorMessage = 'Error: Start date can\'t be later than end date';
       return;
     }
+    if (highlightFormResult.description === null) {
+      this.dateErrorMessage = 'Error: Please enter a description';
+      return;
+    }
```

Comment:
```suggestion
    } else {
      if (highlightFormResult.description.length < 75 || highlightFormResult.description.length > 155)
      {
        this.dateErrorMessage = 'Error: Description must be between 75 and 155 characters long';
        return;
      }
    }
```

Add additional check and make sure that we don't receive too short/long descriptions.